src/pretalx/submission/models/submission.py

```diff
@@ -288,39 +299,70 @@ def _set_state(self, new_state, force=False, person=None):
                 )
             )
 
-    def make_submitted(self, person=None, force=False, orga=False):
-        self._set_state(SubmissionStates.SUBMITTED, force, person=person)
+    def update_talk_slots(self, slot_count_old=None):
```

Comment:
Why would we need `slot_count_old`? We can just check how many slots we currently have vs how many we *should* have, and adjust accordingly.

Reply:
hm i thought this too - but i don't know how to get the original (old) value.
i tried some ways back then but did not get anywhere. 
if you have any hints on this iam open to reworking this :-)

Reply:
done.